app/src/main/java/com/sildev/musicplayer/adapter/SongAdapter.kt

```diff
@@ -0,0 +1,72 @@
+package com.sildev.musicplayer.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.sildev.musicplayer.MusicPlayerHelper.getBitmapSong
+import com.sildev.musicplayer.MusicPlayerHelper.removeAccent
+import com.sildev.musicplayer.R
+import com.sildev.musicplayer.databinding.ItemSongBinding
+import com.sildev.musicplayer.model.Song
+
+
+class SongAdapter(private val iClickSongItem: IClickSongItem) :
+    RecyclerView.Adapter<SongAdapter.SongViewHolder>() {
+    interface IClickSongItem {
+        fun onClickItem(position: Int)
+    }
+
+    private var _listSong: List<Song> = ArrayList()
+    private var resultSongList: MutableList<Song> = ArrayList()
```

Comment:
khởi tạo Array mà lại để kiểu MutableList :))
```suggestion
    private var resultSongList = mutableListOf<Song>()
```